package.json

```diff
@@ -346,12 +346,13 @@
     "@brave/react-virtualized-auto-sizer": "^1.0.4",
     "@glif/filecoin-wallet-provider": "2.0.0-beta.12",
     "@ledgerhq/hw-app-eth": "6.24.1",
+    "@ledgerhq/hw-app-solana": "6.27.1",
     "@ledgerhq/hw-transport-webhid": "6.24.1",
+    "@solana/web3.js": "1.44.2",
```

Comment:
thanks for bumping this

Reply:
Bumped again because I don't think 1.44.2 contained the fix you wanted